### PR TITLE
Moved flask app out of global space, moved routes to a class, updated…

### PR DESCRIPTION
**Pull request by fall-byrd**
… related calls

## TL;DR
Quick summary/list of your change(s)

## What
What is affected by this PR?
- [X] API
- [ ] GUI
- [ ] Hardware Integration
- [ ] OS/Deployment
- [ ] Documentation
- [ ] Other (please describe in notes)

## Testing
How was this tested?
- [X] Locally Virtualized
- [ ] Raspberry Pi
- [ ] With Hardware
- [] Other (please describe in notes)
 
Tested the usual way, unit tests pass and also through the UI.

Figured I'd start with getting the changes in with a relatively small set initially, the routes.py changes look more intimidating than they are: moved the routes to a class to move more out of global space, maybe not required but with the recent difficulty we had the more out of global space the better.

### CODE REVIEW

**Comment by fall-byrd** [comment]
This does not resolve the overall issue I was looking to fix with the improper shutdown but that will be forthcoming shortly in another PR after this is merged in. Trying to keep this in smaller chunks as much as I can.

**Review by mike-kiss — APPROVED** [review]
LGTM
